Fix checkmark broken styles + polishes

juan-cortes opened the pull request:
The styles for the checkmarks were failing in places like add account flow. This fixes that.
Also, we've identified the incorrect usage of the checkmark icon where it should be the full checkmark icon. ~I will address that icon change in a separate pr since this one was aimed at styling the badges mainly~. Addressed.

### Type

UI Polish

gre commented:
i'm not sure why you show this screenshot @juan-cortes  because actually we need to drop this UI

juan-cortes commented:
I fixed the checkmarks, it doesn't matter if that banner is going to be gone now. 
I checked every step with brian and I fixed a bunch of broken UI, no hodl needed.
